draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/MobileActions.scss

```diff
@@ -19,29 +21,48 @@ $mobile-actions-split-select-square-width: 64px;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   overflow: hidden;
+  // $kz-layout-mobile-actions-drawer-height is 60px but we can't use the var in calc()
+  transform: translateY(calc(100% - 60px));
```

Comment:
Yes you can - `calc(100% - #{$kz-layout-mobile-actions-drawer-height})`

Reply:
OMG

Reply:
Damn it, I used to know this